doppler/resource_webhook.go

```diff
@@ -0,0 +1,260 @@
+package doppler
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceWebhook() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceWebhookCreate,
+		ReadContext:   resourceWebhookRead,
+		UpdateContext: resourceWebhookUpdate,
+		DeleteContext: resourceWebhookDelete,
+		Schema: map[string]*schema.Schema{
+			"slug": {
+				Description: "The slug of the Webhook",
+				Type:        schema.TypeString,
+				Computed:    true,
+			},
+			"project": {
+				Description: "The name of the Doppler project where the webhook is located",
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+			},
+			"url": {
+				Description: "The URL of the webhook endpoint",
+				Type:        schema.TypeString,
+				Required:    true,
+			},
+			"enabled": {
+				Description: "Whether the webhook is enabled or disabled.  Default to true.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+			},
+			"secret": {
+				Description: "Secret used for request signing",
+				Type:        schema.TypeString,
+				Optional:    true,
+				Sensitive:   true,
+			},
+			"authentication": {
+				Description: "Authentication method used by the webhook",
+				Type:        schema.TypeList,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"token": {
+							Type:      schema.TypeString,
+							Optional:  true,
+							Sensitive: true,
+						},
+						"username": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"password": {
+							Type:      schema.TypeString,
+							Optional:  true,
+							Sensitive: true,
+						},
+					},
+				},
+				MaxItems: 1,
+				Optional: true,
+			},
+			"payload": {
+				Description: "The webhook's payload as a JSON string.  Leave empty to use the default webhook payload",
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
```

Comment:
We can also remove this one, right?

Reply:
fixed in this [force-push](https://github.com/DopplerHQ/terraform-provider-doppler/compare/b342004a8e6c72aa5be6d225b4ae24b453144ec1..99c350df449fb8b4159f887a059644929d0420d4)